dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -280,18 +279,18 @@ public void testComponent_linkageCheckResult() throws IOException, ParsingExcept
     Nodes reports = document.query("//p[@class='jar-linkage-report']");
     Assert.assertEquals(1, reports.size());
     Truth.assertThat(trimAndCollapseWhiteSpace(reports.get(0).getValue()))
-        .isEqualTo(
-            "106 target classes causing linkage errors referenced from 516 source classes.");
+        .isEqualTo("106 symbols causing linkage errors on 516 references.");
```

Comment:
This phrasing still doesn't quite work. In any case, we should make this change in a separate PR, not as part of the refactoring.